src/panels/Menu/TopMenuBar/TopBarMenuWrapper.tsx

```diff
@@ -1,24 +1,28 @@
 import { PropsWithChildren } from 'react';
 import {
   ArrowPosition,
+  Button,
   FloatingAxesOffsets,
   FloatingPosition,
   Menu,
-  MenuProps
+  MenuProps,
+  Text
 } from '@mantine/core';
 
 interface Props extends MenuProps, PropsWithChildren {
+  targetTitle: string | React.ReactNode;
   menuItemTabIndex?: -1 | 0;
   position?: FloatingPosition;
   offset?: number | FloatingAxesOffsets;
   withArrow?: boolean;
   arrowPosition?: ArrowPosition;
```

Comment:
No need to define these here, they are already in MenuProps

Reply:
Good catch